### PR TITLE
Changes for integrating live share sdk in live-coding c# sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/meetings-live-code-interview/csharp/MeetingLiveCoding/ClientApp/public/index.html [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+  <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description"
-          content="Website created using create-react-app" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (ChetanSharma-msft) [inline samples/meetings-live-code-interview/csharp/MeetingLiveCoding/ClientApp/public/index.html:10]: We can change this content.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my comment

File: samples/meetings-live-code-interview/csharp/README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ sequenceDiagram
 - [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
```
Comment (ChetanSharma-msft) [inline samples/meetings-live-code-interview/csharp/README.md:60]: We can change header/title of readme file as
"This sample demos a live coding in a teams meeting stage using live share SDK."

Also, can add doc links of live share sdk in further reading section.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

File: samples/meetings-live-code-interview/csharp/README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ extensions:
 
 # Live coding interview using Shared meeting stage 
 
-This sample demos live coding in a teams meeting. In side panel there is a list of question in specific coding language and on share click specific question with language code editor will be shared with other participant in meeting.
+This sample demos a live coding in a teams meeting stage using live share SDK. In side panel there is a list of question in specific coding language and on share click specific question with language code editor will be shared with other participant in meeting.
```
Comment (Wajeed-msft) [inline samples/meetings-live-code-interview/csharp/README.md:17]:
```suggestion
This sample demos a live coding in a Teams meeting stage using [Live Share SDK](https://aka.ms/livesharedocs). In side panel there is a list of question in specific coding language and on share click specific question with language code editor will be shared with other participant in meeting.
```

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**